addon/helpers/build-secondary-nav-links.js

```diff
@@ -0,0 +1,83 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+/**
+ * @module ember-osf
+ * @submodule helpers
+ */
+
+/**
+ * Returns secondary links corresponding to a given service
+ *
+ * @class buildSecondaryNavLinks
+ * @param {String} currentService name of current service (HOME, PREPRINTS, REGISTRIES, or MEETINGS)
```

Comment:
Rather than hardcode these constants with special meaning throughout the code, might I suggest defining them as predefined constants in a single shared file that everything else references? It might ease maintainability later.